core/src/test/java/io/questdb/griffin/InsertNullTest.java

```diff
@@ -234,4 +234,18 @@ static String expectedNullInserts(String header, String nullValue, int count) {
         }
         return sb.toString();
     }
+
+    static String expectedNullInserts(String header, String type, String nullValue, int count, boolean eq) {
```

Comment:
I would simply use one method called `expectedNullInserts` like so:

```
static String expectedNullInserts(String header, String nullValue, int count, boolean expectsOutput) {
        StringSink sb = Misc.getThreadLocalBuilder();
        sb.put(header);
        if (expectsOutput) {
            for (int i = 0; i < count; i++) {
                sb.put(nullValue).put("\n");
            }
        }
        return sb.toString();
    }

    private static boolean isNotNullable(String type) {
        return Chars.equalsLowerCaseAscii(type, "short") ||
                Chars.equalsLowerCaseAscii(type, "byte") ||
                Chars.equalsLowerCaseAscii(type, "boolean");
    }
```

then you can call it:

```
testInsertNullThenFilterIsNull: expectedNullInserts("value\n", type[1], NULL_INSERTS, !isNotNullable(type[0])),
testInsertNullThenFilterIsNotNull: expectedNullInserts("value\n", type[1], NULL_INSERTS, isNotNullable(type[0])),
```

Note, you will need to amend InsertNullGeoHashTest (add true at the end in L150)